### PR TITLE
Issue 117.bryan nice

### PR DESCRIPTION
**Pull request by bryan-nice**
# Pull request questions

## Which issue does this address

Issue number: #117 

## Why was change needed

added build test coverage for RHEL 7 distribution

## What does change improve

Provides two distribution build test coverage.

### CODE REVIEW

File: .github/workflows/cd-3hr-cron-test-debian-build.yml [hunk 1]
```diff
@@ -14,7 +14,11 @@ jobs:
       - name: Build Java API Server
         id: build_java_api_server
         run: |
-          docker build --tag sezing-java-api-server:${GITHUB_HEAD_REF:-${GITHUB_REF##*/}} \
+<<<<<<< HEAD
```
Comment (docktermj) [inline .github/workflows/cd-3hr-cron-test-debian-build.yml]: @bryan-nice I don't think this is what you want in Master.   This shows a merge conflict that you'll have to rectify manually.

Comment (bryan-nice) [reply .github/workflows/cd-3hr-cron-test-debian-build.yml]: thanks for catching that. geez I thought I had that resolved. I was testing a forced conflict.

Comment (bryan-nice) [reply .github/workflows/cd-3hr-cron-test-debian-build.yml]: ok corrected. Thank you for the catch

File: .github/workflows/docker/ci-cd-debian-build/Dockerfile [hunk 2]
```diff
@@ -31,13 +31,8 @@ RUN apt-get update \
...
 # Add Senzing Files
-RUN if [[ "${SENZING_REPO_ENV}" == "PRD" ]]; then \
-        SENZING_REPO_URL="https://senzing-production-apt.s3.amazonaws.com/senzingrepo_1.0.0-1_amd64.deb"; \
-    else \
-        SENZING_REPO_URL="https://senzing-staging-apt.s3.amazonaws.com/senzingstagingrepo_1.0.0-1_amd64.deb"; \
-    fi \
-    && wget ${SENZING_REPO_URL} \
-    && apt-get -y install ./senzingstagingrepo_1.0.0-1_amd64.deb \
+RUN wget ${SENZING_REPO_URL} \
+    && apt-get -y install ./${SENZING_REPO_URL##*/} \
```
Comment (docktermj) [inline .github/workflows/docker/ci-cd-debian-build/Dockerfile]: @bryan-nice Just for my education, what does the `##*/` do?

Comment (bryan-nice) [reply .github/workflows/docker/ci-cd-debian-build/Dockerfile]: its leverage the natural parameter expansion to resolve the file name at the end of the URL by only returning the string after the last forward slash /. 

Here is more info on parameter expansion
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Comment (docktermj) [reply .github/workflows/docker/ci-cd-debian-build/Dockerfile]: Neat!  Thanks for the pointer!

**Pull request merged**